SwrveSDK/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Sorry, I thought it did, but I could be wrong. If it doesn't, then disregard! :)